OpenSim/Simulation/RegisterTypes_osimSimulation.cpp

```diff
@@ -273,7 +272,8 @@ OSIMSIMULATION_API void RegisterTypes_osimSimulation()
     Object::renameType("MusclePoint",       "PathPoint");
     Object::renameType("MuscleViaPoint",    "ConditionalPathPoint");
     Object::renameType("MovingMusclePoint", "MovingPathPoint");
-    Object::renameType("MusclePointSet",    "PathPointSet");
+    Object::renameType("MusclePointSet", "PathPointSet");
+    Object::renameType("Appearance", "SurfaceAppearance");
```

Comment:
I am not really familiar with the geometry changes but we could have issues with the fact that there is still a valid Appearance class. People's valid Appearance classes could be silently converted to SurfaceAppearance. If the new Appearance class is in fact virtual, then the issue might go away. Not sure though. 

If this line of code is not changed, then we should at least add a comment so people know that this instance of "Appearance' is referring to what will be a historical intermediate version of the SurfaceAppearance class.